setup_configure.py

```diff
@@ -58,6 +58,7 @@ class EnvironmentOptions(object):
     def __init__(self):
         self.hdf5 = os.environ.get('HDF5_DIR')
         self.hdf5_version = os.environ.get('HDF5_VERSION')
+        self.mpi = os.environ.get('HDF5_PARALLEL') == "ON"
```

Comment:
found [this way](http://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python/715455#715455) but would depend on a pre-defined list of "what shall be true" strings.

do you suggest
- s.toLower() in ["on", "1", "true"] as `True`
- everything else as false

?